### PR TITLE
Add `kubectl-mayastor` 2.5.0.

### PR DESCRIPTION
**Pull request by reitermarkus**
Redo of https://github.com/kubernetes-sigs/krew-index/pull/3465.

### CODE REVIEW

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

Thanks for submitting your kubectl plugin to Krew!
One of the krew-index maintainers will review it soon. Note that the reviews for new plugin submissions may take a few days.

In the meanwhile, here are a few tips to make your plugin manifest better:
* Make sure your plugin follows the [best practices](https://krew.sigs.k8s.io/docs/developer-guide/develop/best-practices/).
* Eliminate redundant wording form `shortDescription` (it should be max 50 characters).
* Try to word wrap your `description` to 80-character lines (no usage examples, please).

Thanks for your patience!
/kind new-plugin

File: plugins/mayastor.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  version: v2.5.0
+  homepage: https://mayastor.gitbook.io/
+  shortDescription: Provides commands for OpenEBS Mayastor.
+  description: |
+    This plugin allows you to manage Mayastor volumes and upgrades.
+  platforms:
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
```
Comment (ahmetb) [inline plugins/mayastor.yaml:15]: You should look into adding darwin/arm64 as that's the only macbooks sold nowadays practically nowadays.

**Comment by ahmetb** [comment]
/lgtm
/approve

Please set up [Krew release automation](https://krew.sigs.k8s.io/docs/developer-guide/release/automating-updates/) which helps you skip manually making updates to your Krew manifests on each new version and send a pull request. It’s a GitHub workflow bot that can run every time you push a new tag. Those PRs are automatically approved.

**Comment by reitermarkus** [comment]
CC @tiagolobocastro regarding macOS ARM binaries and release automation.

**Comment by tiagolobocastro** [comment]
Thanks for this @reitermarkus, just installed mayastor plugin via krew and it works :)

> CC @tiagolobocastro regarding macOS ARM binaries and release automation.
Yeah, we don't have any macOS ARM runners at the moment so it'll be difficult to build. We might be able to cross build from macOS x86, we were waiting for some cross-compiling fixes on Nixpkgs, I'll give that a try soon and see if it's been resolved.

**Pull request merged**